packages/dappmanager/src/modules/eth2migration/export/exportKeys.ts

```diff
@@ -0,0 +1,48 @@
+import { Eth2Network, eth2migrationParams } from "../params";
+import shell from "../../../utils/shell";
+import { parseValidatorAccounts } from "./utils";
+
+/** Export validator public keys from Prysm-non-web3sgigner version */
+export async function exportValidatorKeys({
+  network,
+  containerName
+}: {
+  network: Eth2Network;
+  containerName: string;
+}): Promise<void> {
+  try {
+    // 1. List keys
+    // - Example command: validator accounts list --wallet-dir=/root/.eth2validators --wallet-password-file=/root/.eth2validators/walletpassword.txt --prater
+    const validatorAccountsData = await shell(
+      `docker exec ${containerName} validator accounts list --wallet-dir=${eth2migrationParams.keys.walletDir} \
+--wallet-password-file=${eth2migrationParams.keys.walletPasswordFile} --${network} --accept-terms-of-use`
+    );
+
+    // Get public keys in a string comma separated
+    const validatorAccountsDataParsed = parseValidatorAccounts(
+      validatorAccountsData
+    );
+
+    // 2. Copy walletpassowrd to backup folder
+    shell(
+      `docker exec ${containerName} cp ${eth2migrationParams.keys.walletPasswordFile} ${eth2migrationParams.backup.backupDir}`
+    ).catch(e => {
+      throw Error(`Eth2 migration: backup wallet password failed. ${e}`);
```

Comment:
Don't interpolate Error objects in a template literal. Extend the error message and re-throw